### ISSUE TITLE
Color Contrast Issues don't seem real for this site

### ISSUE THREAD

**Issue by goodwitch** [dequelabs/axe-core#1196]
Oh awesome and fabulous aXe-core team,

I just ran aXe on this page https://www.bang-olufsen.com/en/beosound-edge and got 31 color contrast issues.  When I looked at the details of the first few issues...I see that the text foreground/background colors reported by aXe do not match the reality of what I'm seeing on screen.

Example:  First contrast issue has this specific info:
"Fix the following:
Element has insufficient color contrast of 3.94 (foreground color: #ffffff, background color: #808080, font size: 8.3pt, font weight: normal). Expected contrast ratio of 4.5:1"

But visually, the colors I'm seeing are white #ffffff text (we agree) on a dark green background #1B4B52 that has a contrast ratio of 9.7 to 1.  

The code that is reported as having this "contrast" error is:
<span class="navbar__label navbar__label--left smallest text-uppercase light" style="font-size: 11px !important; line-height: 16.5px !important;">Products</span>

Expected Behavior:  aXe should not have reported this as a color contrast issue...because the white text on the dark green background meets and exceeds the requirements. 

At least, that is what I think from what I'm seeing.  

axe-devtools: 3.5.2
axe-core version: 3.1.1
Mac/Chrome

**Comment by WilcoFiers** [comment]
Good find. There are multiple things going wrong here.

- pointer-events are used in the page for the backgrounds. Elements with `pointer-events:none` can't be tested with axe-core for color contrast. I've never seen it go wrong, but it was bound to come up at some point

- Even with pointer-events enabled, something is still off. It seems like canvas isn't being ignored as it should, or maybe something else is going on here, I'm not entirely sure about it. We'd need to do more research.

**Comment by marcysutton** [comment]
There's a gradient on that canvas element, so shouldn't that item be reported as "incomplete"?

**Comment by WilcoFiers** [comment]
Unfortunately the issue has gone out of date and can no longer be reproduced. Closing this.

**Comment by goodwitch** [comment]
@wilco can you use the WayBack machine to reproduce this?  https://web.archive.org/web/20181121065157/https://www.bang-olufsen.com/en/speakers/beosound-edge
